### PR TITLE
Unpin dask[-expr] and distributed for 24.12 development

### PR DESCRIPTION
**Pull request by pentschev**
Unpins dask, dask-expr and distributed for 24.12 development.

### CODE REVIEW

**Comment by pentschev** [comment]
@rjzamora can you confirm it's ok for us to unpin dask-expr too?

File: pyproject.toml [hunk 1]
```diff
@@ -12,9 +12,9 @@ name = "rapids-dask-dependency"
 version = "24.12.00a0"
 description = "Dask and Distributed version pinning for RAPIDS"
 dependencies = [
-    "dask==2024.9.0",
-    "distributed==2024.9.0",
-    "dask-expr==1.1.14",
+    "dask @ git+https://github.com/dask/dask.git@main",
+    "distributed @ git+https://github.com/dask/distributed.git@main",
+    "dask-expr @ git+https://github.com/dask/distributed.git@main",
```
[the remark this answers is not in the document]
Comment (pentschev) [reply pyproject.toml]: Damn you copy-paste for not automatically adapting the line accordingly!

**Review by galipremsagar — APPROVED** [review]

**Comment by rjzamora** [comment]
>can you confirm it's ok for us to unpin dask-expr too?

Rebuilding my env locally with latest dask/dask-expr/etc.. I'll merge this once I confirm that tests pass locally at least.

**Review by rjzamora — APPROVED** [review]

**Comment by pentschev** [comment]
Thanks @rjzamora and @galipremsagar !

**Pull request closed without merging**